tests/test_loky_backend.py

```diff
@@ -480,38 +477,42 @@ def _check_fds(self, pid, w):
 
         n_pipe = 0
         named_sem = []
+        unexpected = []
         for fd, t, name in zip(lines[::3], lines[1::3], lines[2::3]):
+            # Check if fd is a standard IO file.
+            is_std = fd in ["f0", "f1", "f2"]
 
-            # Check if fd is a standard IO file. For python 3.x stdin
-            # should be closed.
-            is_std = fd in ["f1", "f2"]
-
-            # Check if fd is a pipe
+            # Check if fd is a pipe.
             is_pipe = t in ["tPIPE", "tFIFO"]
             n_pipe += is_pipe
 
             # Check if fd is open for the rng. This can happen on different
-            # plateform and depending of the python version.
+            # platform and depending of the python version.
             is_rng = name == "n/dev/urandom"
 
+            # Not sure if this really expected to be open or not...
+            is_dev_null = name == "n/dev/null"
```

Comment:
I am not 100% sure why this is needed with the new code, and it wasn't before, but keeping an open fd on `/dev/null` sounds harmless.

Reply:
BTW, when editing the assertion below, I checked that there was only one open fd to `/dev/null` and not thousands.